src/pyfmi/fmi.pxd

```diff
@@ -41,9 +41,12 @@ cdef class ModelBase:
     cdef char* _fmu_log_name
     cdef FMIL.jm_callbacks callbacks
     cdef public dict cache
+    cdef public object _log_stream
     cdef public object file_object
     cdef public object _additional_logger
     cdef public object _max_log_size_msg_sent
+    cdef object _modelid
+    cdef public int log_is_stream, _invoked_dealloc
```

Comment:
I suggest to make "log_is_stream" to an internal variable i.e: "_log_is_stream"

Reply:
I agree, it has been fixed.